ruby/alias_manager.rb

```diff
@@ -0,0 +1,24 @@
+#Swap the first and last name
+
+#Change all vowels (a,e,i,o,u) to the next vowel in 'aeiou'. Change all 
+#consonants to the next consonant.
+
+attempts = 0
+
+aliases = []
+
+until attempts == 3
+	attempts += 1
+
+puts "Please list your full name (first and last name)."
+full_name = gets.chomp.downcase!
+
+full_name.tr!('aeiou', 'eioua') 
+full_name.tr!('bcdfghjklmnpqrstvwxyz','cdfghjklmnpqrstvwxyzb')
+
```

Comment:
One of the purposes of the challenge was to have you practice iterating over a data structure, a string, and changing it.  Try using iteration to iterate through the full name string, determining what each character is (vowel or consonant), and determining what it should be changed to.